after/syntax/gitlab.vim

```diff
@@ -2,7 +2,7 @@ if filereadable($VIMRUNTIME . '/syntax/markdown.vim')
   source $VIMRUNTIME/syntax/markdown.vim
 endif
 
-syntax match Username "@\w\+"
+syntax match Username "@\I\+[\_\-\.]*\I\+"
```

Comment:
What does the `\I` refer to here?

For me this isn't capturing the final digit in my username: 

![Screenshot 2023-11-08 at 12 52 27 PM](https://github.com/harrisoncramer/gitlab.nvim/assets/32515581/caa83e4f-3c88-458b-b4a1-7824e484fa14)

Reply:
FIxed